testng-eclipse-plugin/src/main/org/testng/eclipse/refactoring/ConvertToYamlAction.java

```diff
@@ -65,6 +65,10 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
             TestNGPlugin.log(e1);
           } catch (CoreException e) {
             TestNGPlugin.log(e);
+          } catch (ParserConfigurationException e) {
```

Comment:
Yeah, it's weird... this worked for me inside a mock build for F24 and F25, and worked in Eclipse using PDE to build automatically. But failed your travis build. So... maybe you're right - a generic Exception catch might be the better solution. I'll concoct a different patch and submit that instead.